08.decoding.process.md

```diff
@@ -6612,7 +6647,25 @@ Other applications
 may set their own policy about which frames are output.
 {:.alert .alert-info }
 
-The arrays OutY, OutU, and OutV are prepared and then film grain noise is added if required.
+The intermediate output preparation process specified in [section 7.18.2][] is invoked to prepare arrays OutY, OutU, and OutV, and the outputs are assigned to w, h, subX, and subY.
+
+If film_grain_params_present is equal to 1 and apply_grain is equal to 1, then
+the film grain synthesis process specified in [section 7.18.3][] is invoked with inputs of w, h, subX, and subY.
+(This process modifies the output arrays OutY, OutU, OutV).
+
+Finally, the frame to be output is defined to be the arrays OutY, OutU, OutV where the bit depth for each sample is BitDepth.
+
+This frame to be output is the overall output of the decoding process and further processing (such as color conversion) is outside the scope of this specification.
+
+For example, a real implementation might use these arrays to display the frame to the user, or a test system might save the arrays so the output can be verified.
+
+**Note:** If NumPlanes is equal to 1, then the U and V planes should be ignored.
+{:.alert .alert-info }
+
+##### Intermediate output preparation process
```

Comment:
I think this should only have 4 # characters now (i.e. the same as for film grain synthesis process)